components/alert/src/Alert.js

```diff
@@ -17,17 +17,18 @@ export const Alert = ({ appearance, closable, icon: Icon, children }) => {
 	const [open, setOpen] = useState(true);
 	const { breakpoints, alert } = useTheme();
 	const mq = paint(breakpoints);
+	const getPaddingSM = () =>
+		closable ? `${alert.padding} 30px ${alert.padding} ${alert.padding}` : alert.padding;
 
 	const styleCommon = {
 		marginBottom: alert.marginBottom,
-		padding: alert.padding,
-		paddingRight: [18, 30],
+		padding: getPaddingSM(),
```

Comment:
^ We need to disregard the extra RHS padding in XS; it doesn't matter about space for the button in XS since it's floating. We're better to make use of all the space we have for the text.